Sources/Containerization/Image/ImageStore/ImageStore.swift

```diff
@@ -22,29 +22,66 @@ import Foundation
 /// An ImageStore handles the mappings between an image's
 /// reference and the underlying descriptor inside of a content store.
 public actor ImageStore: Sendable {
+    /// The ImageStore path it was created with.
+    public nonisolated let path: URL
+
     private let referenceManager: ReferenceManager
     internal let contentStore: ContentStore
     internal let lock: AsyncLock = AsyncLock()
 
-    public init(path: URL, contentStore: ContentStore) throws {
+    public init(path: URL, contentStore: ContentStore? = nil) throws {
         try FileManager.default.createDirectory(at: path, withIntermediateDirectories: true)
 
-        self.contentStore = contentStore
+        if let contentStore {
+            self.contentStore = contentStore
+        } else {
+            self.contentStore = try LocalContentStore(path: path.appendingPathComponent("content"))
+        }
+
+        self.path = path
         self.referenceManager = try ReferenceManager(path: path)
     }
+
+    /// Return the default image store for the current user.
+    public static var `default`: ImageStore {
+        get throws {
+            let root = Self.defaultRoot()
+            return try Self(path: root)
+        }
+    }
+
+    private static func defaultRoot() -> URL {
+        let root = FileManager.default.urls(
+            for: .applicationSupportDirectory,
+            in: .userDomainMask
+        ).first
+        guard let root else {
+            fatalError("unable to get Application Support directory for current user")
```

Comment:
I'm usually not a huge fan of panic'ing in libraries, but this one seems somewhat harmless. However, this is private and the place we call it from also is marked as throws, so can we just bubble up an error instead?